eidpauth/src/main/java/uk/ac/ucl/eidp/auth/jaas/EidpLoginModule.java

```diff
@@ -59,7 +63,7 @@ private void setUserService() {
                 CreationalContext cc = beanManager.createCreationalContext(bean);
                 userService = (UserController) beanManager.getReference(bean, UserController.class, cc);
             } catch (NamingException e) {
-                throw new RuntimeException(e);
+                logger.log(Level.SEVERE, "Cannot call UserController bean", e);
```

Comment:
\[Checkstyle\] WARNING: 'catch' child have incorrect indentation level 16, expected level should be 8\.